src/storage/src/generated/protos/storage/google.storage.v2.rs

```diff
@@ -172,9 +172,7 @@ pub mod compose_object_request {
         #[prost(int64, tag = "2")]
         pub generation: i64,
         #[prost(message, optional, tag = "3")]
-        pub object_preconditions: ::core::option::Option<
-            source_object::ObjectPreconditions,
-        >,
+        pub object_preconditions: ::core::option::Option<source_object::ObjectPreconditions>,
```

Comment:
This is generated code. In particular, this is generated by Prost. Please revert the changes to this file.